### ISSUE TITLE
Update accessibility metadata guidance

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2626]
There have been a few changes to the accessibility metadata since we last published the techniques. We need to update the document to note:

- that there are now individual unknown hazard values
- that the `pageNavigation` feature should be used when a page list is included
- that the `pageBreakMarkers` feature replaces `printPageNumbers`
- that the `a11y:pageBreakSource` property should now be used instead of the `source-of` refinement property.
